docs/source/ops.rst

```diff
@@ -10,49 +10,98 @@ Operators
 .. note::
   All operators have native support for TorchScript.
 
```

Comment:
On line 8. Should we add more comprehensive summary?

Reply:
Yes that's worth it. Perhaps mention that this included layers, losses etc.